writeToStream.js

```diff
@@ -0,0 +1,579 @@
+
+'use strict';
+
+var protocol = require('./constants')
+  , empty = new Buffer(0)
+  , zeroBuf = new Buffer([0])
+  , numCache = require('./numbers')
+
+function generate(packet, stream) {
+
```

Comment:
Exactly, the main reason I've left it out is for backward compatibility, and because I would like the other modules using this to do their cork/uncork as they see fit. Possibly the `process.nextTick` has some disadvantages that I am unaware off.